src/logic/operate.js

```diff
@@ -0,0 +1,22 @@
+import Big from 'big';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const a = new Big(numberOne);
+  const b = new Big(numberTwo);
+
+  if (operation === '-') {
+    return a.minus(b);
+  }
+
+  if (operation === '+') {
+    return a.plus(b);
+  }
+
+  if (operation === 'x') {
+    return a.mul(b);
+  }
+
+  return a.div(b);
+};
+
+export default operate;
```

Comment:
- Here we are missing one operation. The one you get with the % sign 
![image](https://user-images.githubusercontent.com/46115725/80282409-e0270b80-86d6-11ea-91ad-5aa41f35fae8.png)
-  The % sign should not be confused with the modulo operator and should use the modifier property turning the number into a percentage based number e.g. `95% == .95`.